src/components/Container/Container.tsx

```diff
@@ -0,0 +1,12 @@
+import styles from "./Container.module.scss";
+
+interface Content {
```

Comment:
```suggestion
interface IProps {
```

When we have interface that is specify for the props and it is not exported outside of the file, we can name it Prop. Add 'I' before the name is a good practice for naming interfaces overall.

Reply:
Add this chnge to other files as well.